test/integration/targets/sqitch/tasks/test.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Deploy in check mode with cwd
+  sqitch: command=deploy cwd={{ output_dir }} plan_file=sqitch.plan target=anstest verify=true to_change=types
+  check_mode: yes
+
+- name: Deploy in check mode without cwd
+  sqitch: command=deploy plan_file=sqitch.plan target=anstest verify=true to_change=types
+  check_mode: yes
+
+- name: Deploy to change
+  sqitch: command=deploy cwd={{ output_dir }} plan_file=sqitch.plan target=anstest verify=true to_change=types
```

Comment:
Please write tasks using `key: value` syntax (as was done below) instead of `key=value` single line syntax.